hello-jni/app/build.gradle

```diff
@@ -58,16 +52,10 @@ android {
                 abiFilter 'x86_64'
             }
         }
-        mips {
-            dimension 'cpuArch'
-            ndk {
-                abiFilters 'mips', 'mips64'
-            }
-        }
         universal {
             dimension 'cpuArch'
             ndk {
-                abiFilters 'mips', 'mips64', 'x86', 'x86_64'
+                abiFilters 'x86', 'x86_64'
```

Comment:
What's this? How is it universal if it only targets half of the architectures?

Reply:
this is just an example to show that application could pick up any kinds of combination of the ABI into flavor build.

Reply:
I still don't understand how this is in any way "universal", but it's not related to the rest of the patch, so whatever. Should probably add some comments explaining why this is the way it is (I sure don't understand it and as far as gradle knowledge goes, I am the target user of the samples), but that can be a different patch.

Reply:
yeah, will look at that with next patch for the compiler and stl default handling.